### PR TITLE
Trots/task1

### PR DESCRIPTION
**Pull request by lost106**

### CODE REVIEW

File: Tasks/Trots/Task1/Task1.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from decimal import Decimal
+from decimal import getcontext
+
+getcontext().prec = 50
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Trots/Task1/Task1.py:4]: this is not necessary, but a nice touch

File: Tasks/Trots/Task1/Task1.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+start_deposit = Decimal('20000')
+rate = Decimal('0.15')
+term = Decimal('60')
+
+final_sum = start_deposit*(Decimal('1') + rate/Decimal('12'))**term
+
+print("The amount of the deposit at the end of the term will be - ", final_sum)
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Trots/Task1/Task1.py:12]: and here we get a 50 decimal digits based answer) would be nice to round it to get some realistic money related answer (like with some cents)

**Review by MikitaTsiarentsyeu — APPROVED** [review]
Good job, the score is 8.5

File: Tasks/Trots/Task1/Task1.py (hunk 2 above)
Comment (lost106) [reply Tasks/Trots/Task1/Task1.py:12]: i just wanted to show the difference between using decimals and normal numbers so i didn't use round

**Pull request merged**